regression-tests/test-results/msvc-2022-c++20/run-tests-msvc-2022.bat

```diff
@@ -1,9 +1,9 @@
-REM This is intended to be run in the /test-results/msvc-2022 directory
+REM This is intended to be run in the /test-results/msvc-2022-c++20 directory
 REM in a VS2022 Command Prompt (which sets the path and other environment)
 REM
 REM After installing VS2022, to finish setting up std modules in your environment, run:
-REM     cl /std:c++latest /EHsc /nologo /W4 /MD /c "%VCToolsInstallDir%\modules\std.ixx"
-REM     cl /std:c++latest /EHsc /nologo /W4 /MD /c "%VCToolsInstallDir%\modules\std.compat.ixx"
+REM     cl /std:c++20 /EHsc /nologo /W4 /MD /c "%VCToolsInstallDir%\modules\std.ixx"
+REM     cl /std:c++20 /EHsc /nologo /W4 /MD /c "%VCToolsInstallDir%\modules\std.compat.ixx"
```

Comment:
I overlooked this change in my first review. It is OK with me, but @hsutter please confirm this is something you want. I guess you use this script a lot, don't you?

Reply:
That change is in the new msvc `c++20` directory. The original bat file is unchanged but its directory now has `c++latest` in its name.

Reply:
Sorry, my bad.

Reply:
Sorry, my bad.